src/test/scala/org/zalando/nakadi/client/KlientSpec.scala

```diff
@@ -515,3 +485,53 @@ class KlientSpec extends WordSpec with Matchers with BeforeAndAfterEach with Laz
     service.start()
   }
 }
+
+object KlientSpec extends WordSpecLike /*for 'info()'*/ with ShouldMatchers {
```

Comment:
One can pull aside testing tools to a companion object (so people can start reading the tests, early on in the file). By deriving from `WordSpecLike` we get `info` for providing additional data about particular tests (println-like, but slightly different - see below).